### PR TITLE
NE-675: add saga for delete dvd

### PR DESCRIPTION
**Pull request by HennieScheepersNeslo**

### CODE REVIEW

File: src/state/dvd/dvd.saga.js [hunk 1]
```diff
@@ -66,8 +68,27 @@ function* watchForEditDVD() {
   yield takeLatest(editDVD.type, editDVDSaga);
 }
 
+function* removeDVDSaga(action) {
```
Comment (noydb) [inline src/state/dvd/dvd.saga.js:71]: destructure action pls

**Review by noydb — CHANGES_REQUESTED** [review]

**Review by noydb — APPROVED** [review]

File: src/state/dvd/dvd.saga.js [hunk 2]
```diff
@@ -68,16 +68,16 @@ function* watchForEditDVD() {
...
     const {endpoint, axiosOptions} = new ApiRequest(
-      `${API_HOME}/${action.payload}`,
+      `${API_HOME}/${payload}`,
       HttpVerb.DELETE,
       HEADERS
     )
 
     yield call(axios, endpoint, axiosOptions);
-    yield put(deleteDVD(action.payload))
+    yield put(deleteDVD(payload))
```
Comment (noydb) [inline src/state/dvd/dvd.saga.js:80]: can we not just reload the DVDs here

Comment (HennieScheepersNeslo) [reply src/state/dvd/dvd.saga.js:80]: Yeah, that should work. We can probably do it after each action, to be honest. That should clean up the reducers significantly as well. Would you like me to implement it Benj?

Comment (noydb) [reply src/state/dvd/dvd.saga.js]: Yes please

Comment (noydb) [reply src/state/dvd/dvd.saga.js]: sorry for the delay

Comment (HennieScheepersNeslo) [reply src/state/dvd/dvd.saga.js]: Done ✔️

**Review by noydb — APPROVED** [review]

**Pull request merged**